Blueprints/using-powershell-in-packages-and-scripts.md

```diff
@@ -0,0 +1,24 @@
+{{{
+  "title": "Using PowerShell in Packages and Scripts",
+  "date": "5-31-2015",
+  "author": "Ted Henley",
+  "attachments": [],
+  "contentIsHTML": false
+}}}
+
+### Related
+* [Understanding the Difference Between Templates, Blueprints and Packages](https://www.centurylinkcloud.com/knowledge-base/blueprints/understanding-the-difference-between-templates-blueprints-and-packages/)
+* [Building Windows 2012 Script Packages](https://www.centurylinkcloud.com/knowledge-base/blueprints/building-windows-2012-script-packages/)
+
+
+### Overview
+
+When creating packages and scripts, there are many different options.  This article offers some guidance on using PowerShell as your scripting language.
+
+* Be sure to set the execution mode to PowerShell in the package.manifest.
+* Avoid use of console commands, such as Write-Host.  These will cause your scripts to fail when run in a script or package.
+* Scripts are executed from a temporary directory.  Any files you output, such as logs should be written to a directory in another directory.  Alternatively, you can set a flag in the package to [make the directory permanent](https://www.centurylinkcloud.com/knowledge-base/blueprints/blueprints-script-and-software-package-management/).
```

Comment:
when linking to other article in the KB us relative links please. in this case `blueprints-script-and-software-package-management.md` as the link and our import process will fix them up. 

https://github.com/centurylinkcloud/PublicKB#links-kb-article-to-kb-article